README.docker.md

```diff
@@ -0,0 +1,20 @@
+## Setup
+
+Grab some sample data, for exmaple: https://github.com/hathitrust/imgsrv-sample-data
+
+`imgsrv-sample-data` should be in the same directory as `imgsrv`.
+
+## Running
+
+The docker setup has the imgsrv container running `startup_imgsrv` as a FastCGI service on port 31028.
+
+Use a webserver for FastCGI like https://github.com/beberlei/fastcgi-serve to access the FastCGI app:
```

Comment:
It would probably be worthwhile for the purposes of development to optionally start fastcgi-serve in a separate container in `docker-compose.yml`

Reply:
At that point, I'd imagine this http container should be able to run the CGI scripts...

Reply:
It wouldn't be too hard to come up with a basic apache config (which also lets us mock Shibboleth to some extent). Doesn't need to be on this PR. I'm pretty sure I had done this at some point for something else but I'm having a hard time finding where it ended up (might never have been merged)